Change to detect Aboot flash with `flashrom -p internal` command when mtd devices are present
=============================================================================================
flashrom’s `internal programmer` option is used for BIOS flash. With the `internal` programmer option,
flashrom treats the first mtd device to be the internal device and doesn’t probe other flash devices.
Hence, Aboot flash goes undetected.

This change removes mtd device setup from internal programmer init() function.
mtd devices can still be detected with `flashrom -p linux_mtd:dev=x` option.

diff -Naru ../original/flashrom-v1.2.1/internal.c flashrom-v1.2.1/internal.c
--- ../original/flashrom-v1.2.1/internal.c	2022-09-29 10:06:21.000000000 -0700
+++ flashrom-v1.2.1/internal.c	2023-11-09 20:27:39.340349553 -0800
@@ -228,11 +228,6 @@
 	 */
 	internal_buses_supported = BUS_NONSPI;
 
-	if (try_mtd() == 0) {
-		ret = 0;
-		goto internal_init_exit;
-	}
-
 	/* Initialize PCI access for flash enables */
 	if (pci_init_common() != 0) {
 		ret = 1;
